include/event2/util.h

```diff
@@ -63,6 +63,7 @@ extern "C" {
 
 #ifdef _WIN32
 #include <winsock2.h>
+#include <winerror.h>
```

Comment:
Hm, why it is here, not in `evutil.c` ?

Reply:
If ony in `evutil.c`, another error will occurr in `arc4random.c` :
```sh
In file included from d:/libevent-master/evutil_rand.c:134:0:
d:\libevent-master\arc4random.c: In function 'arc4_seed_win32':
d:\libevent-master\arc4random.c:156:33: error: 'NTE_BAD_KEYSET' undeclared (firs
t use in this function)
    if (GetLastError() != (DWORD)NTE_BAD_KEYSET)
                                 ^~~~~~~~~~~~~~
d:\libevent-master\arc4random.c:156:33: note: each undeclared identifier is repo
rted only once for each function it appears in
make[2]: *** [CMakeFiles/event_static.dir/evutil_rand.c.obj] Error 1
make[1]: *** [CMakeFiles/event_static.dir/all] Error 2
make: *** [all] Error 2
```
When include this header in `arc4random.c`, the similar error will occurr in `listener.c`
```sh
d:/libevent-master/listener.c: In function 'start_accepting':
d:/libevent-master/listener.c:580:16: error: 'ERROR_IO_PENDING' undeclared (firs
t use in this function)
   if (error != ERROR_IO_PENDING) {
                ^~~~~~~~~~~~~~~~
d:/libevent-master/listener.c:580:16: note: each undeclared identifier is report
ed only once for each function it appears in
make[2]: *** [CMakeFiles/event_static.dir/listener.c.obj] Error 1
make[1]: *** [CMakeFiles/event_static.dir/all] Error 2
make: *** [all] Error 2
```
And so on...

If only include in `util.h`, all errors will disappear.

Reply:
I would say that this header should be included in all *.c modules, since in general it is not a good idea to export system-wide headers library headers (if this can be avoided, and this can, if *.h do  not requires it)

Reply:
Thanks for your reminder,  I add this header in all c files.